pythonforandroid/graph.py

```diff
@@ -39,10 +39,11 @@ def conflicts(self):
         return False
 
 
-def get_dependency_tuple_list_for_recipe(recipe, blacklist=[]):
+def get_dependency_tuple_list_for_recipe(recipe, blacklist=set()):
```

Comment:
I keep thinking that I hate default mutable args and it should be a practice we want to avoid as much as possible

Reply:
Yes. Please. `blacklist=None`. Add the two lines of code in the function body to avoid the insidious bug down the road...

Reply:
done :+1: